ChangeLog.md

```diff
@@ -39,18 +40,18 @@ Changes listed here are checked in to GitHub ("master" branch unless specificall
 * Bugfix: `long_soft_event` method did not work correctly.
 * Bugfix: Removed attempted optimization by inlining analogReadEnh; I had wrongly assumed that it would then not inline the call to `_analogReadEnh`. Well, it wound up inlining it, with the result being exactly what I had hoped to prevent. I don't think this made it into a release.
 * Bugfix: Event was not functioning correctly on tinyAVR parts.
-* Bugfix: Correct bug when waking from sleep mode via TWI address match (Thanks @MX682X. You are one of our MVPs)
+* Bugfix: Correct bug when waking from sleep mode via TWI address match (Thanks @MX682X. You are one of our MVPs) - sleepy TWI slaves now work reliably in all sleep modes.
 * Bugfix: 3224 optiboot maximum upload size was same as 1624. Oops.
 * Bugfix: Microchip boards in optiboot mode (admittedly a rare mode) were not subtracting the size of the bootloader from the chip capacity.
 * Enhancement: Lay groundwork in Event library for the new event system changes in the EA-series (There will be 2 generators per port, and 2 for RTC - but all of these will be accessible by all event channels, and a register on the peripheral controls which of the options is used for these two channels). While inapplicable to tinyAVR, Event.h and Event.cpp distributed with DxCore and megaTinyCore is identical.
 * Bugfix: Fix compatibility of Comparator, Event, and Logic.
-* Workaround: mysterious issues with detecting the major version.
-* Major enhancement: Change class hierarchy for UARTs, as was done for Two_Wire (Wire.h), so that rather than pulling in api/HardwareSerial.h, and subclassing that definition of HardwareSerial (itself a subclass of Stream) as UartClass, we instead simply subclass Stream directly. UART.h will be renamed to HardwareSerial.h, HardwareSerial.h (a compatibility layer) will be renamed to UART.h and the latter adjusted to #define UartClass as HardwareSerial, and api/HardwareSerial.h will be gutted and simply #include <HardwareSerial.h) This will remove yet another piece of the disastrous "ArduinoAPI". I describe it as such because on low resource platforms like AVR; this will allow several functions currently declared virtual to lose that keyword, as the linker, even with LTO enabled, is not permitted to remove unused virtual functions. This reduces binary size when using serial significantly, despite significant new features. Credit goes to @MX624X.
+* Workaround: mysterious issues with detecting the major version that prevented manual installation in 1.8.19.
```

Comment:
I'll fix this after merge, but it actually prevented manual installation in 1.8.14+.

Reply:
You are more correct. But IMO the only version post 1.8.13 anyone cares about is the latest one and since I doubt there will be another release from that branch I figured I would reference the version people might care about